addons/website_sale/views/sale_order_views.xml

```diff
@@ -7,10 +7,33 @@
         <field name="arch" type="xml">
             <search>
                 <filter string="Confirmed Orders" name="order_confirmed" domain="[('state', 'in', ('sale', 'done'))]"/>
+                <separator/>
+                <filter string="From Website" name="from_website" domain="[('team_id.team_type', '=', 'website')]"/>
+                <separator/>
+                <filter string="Recovery Email to Send" name="recovery_email" domain="[('cart_recovery_email_sent', '=', False)]" />
             </search>
         </field>
     </record>
 
+    <record id="sale_order_view_form_cart_recovery" model="ir.ui.view">
+        <field name="name">sale.order.form.acquirer</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='team_id']" position="after">
+                <field name="is_abandoned_cart" invisible="1"/>
+                <field name="cart_recovery_email_sent" invisible="1"/>
+            </xpath>
+            <xpath expr="//button[@name='action_quotation_send' and @states='draft']" position="attributes">
```

Comment:
I would change the view_order_form view to remove the "states=draft" and use a real invisible attribute. Then it will be easier to override here because it is based on some implementation details of the web client.